### PR TITLE
Just a bunch of fixups

### PR DESCRIPTION
**Pull request by Nekotekina**

### CODE REVIEW

**Comment by Nekotekina** [comment]
Removed last commit for now, can you test again?

File: rpcs3/Emu/Memory/vm.cpp [hunk 1]
```diff
@@ -270,16 +270,26 @@ namespace vm
...
+
+		// Limit to 256 MiB at once; make sure if it operates on big amount of data, it's page-aligned
+		if (size > 256 * 1024 * 1024 || (size > 65536 && size % 4096))
 		{
 			fmt::throw_exception("Failed to lock range (flags=0x%x, addr=0x%x, size=0x%x)" HERE, flags >> 32, addr, size);
 		}
 
 		// Block or signal new range locks
 		g_range_lock = addr | u64{size} << 35 | flags;
+
```
Comment (Megamouse) [inline rpcs3/Emu/Memory/vm.cpp:291]: ?

Comment (Nekotekina) [reply rpcs3/Emu/Memory/vm.cpp:291]: Currently irrelevant, but will be needed later.

**Comment by Nekotekina** [comment]
Pushed another version, that worked before for some people

**Pull request merged**